modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -523,6 +533,66 @@ protected function register_controls() {
 		] );
 
 		$this->end_controls_section();
+
+		$this->start_controls_section( 'section_box_style', [
+			'label' => esc_html__( 'Box', 'elementor' ),
+			'tab' => Controls_Manager::TAB_STYLE,
+		] );
+
+		$this->add_group_control(
+			Group_Control_Background::get_type(),
+			[
+				'name' => 'box_background_color',
+				'types' => [ 'classic', 'gradient' ],
+				'exclude' => [ 'image' ],
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+			]
+		);
+
+		$this->add_group_control(
+			Group_Control_Border::get_type(),
+			[
+				'name' => 'box_border',
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+			]
+		);
+
+		$this->add_responsive_control(
+			'box_border_radius',
+			[
+				'label' => esc_html__( 'Border Radius', 'elementor' ),
+				'type' => Controls_Manager::DIMENSIONS,
+				'size_units' => [ 'px', 'em', 'rem' ],
+				'selectors' => [
+					'{{WRAPPER}}' => '--tabs-v2-content-wrapper-border-radius: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				],
+			]
+		);
+
+		$this->add_group_control(
+			Group_Control_Box_Shadow::get_type(),
+			[
+				'name' => 'box_shadow_box_shadow',
+				'selector' => '{{WRAPPER}} .elementor-tabs-content-wrapper',
+				'condition' => [
+					'box_height!' => 'height',
+				],
+			]
+		);
+
+		$this->add_responsive_control(
+			'box_padding',
```

Comment:
I tried to test the padding control and it didn't work for me.
Can you double check it?

Reply:
See above

Reply:
Re-tested, works.